.gitlab-ci.yml

```diff
@@ -0,0 +1,51 @@
+stages:
+  - deploy_to_sonatype
+  - create_key
+
+variables:
+  REGISTRY: 486234852809.dkr.ecr.us-east-1.amazonaws.com
+
+# From the tagged repo, push the release artifact
+deploy_to_sonatype:
+  stage: deploy_to_sonatype
+
+  rules:
+    # Build tags automatically
+    - if: '$CI_COMMIT_BRANCH == "master" && $CI_COMMIT_TAG =~ /^\d+.\d+.\d+/'
+
+    # Allow manual triggering of the pipeline
+    - when: manual
+      allow_failure: true
+
+  tags:
+    - "runner:docker"
+    - "size:large"
+
+  image: maven:3.6.3-jdk-8-slim
+
+  script:
+    - export SONATYPE_USER=$(aws ssm get-parameter --region us-east-1 --name ci.jmxfetch.publishing.sonatype_username --with-decryption --query "Parameter.Value" --out text)
+    - export SONATYPE_PASS=$(aws ssm get-parameter --region us-east-1 --name ci.jmxfetch.publishing.sonatype_password --with-decryption --query "Parameter.Value" --out text)
+    - export GPG_PASSPHRASE=$(aws ssm get-parameter --region us-east-1 --name ci.jmxfetch.signing.gpg_passphrase --with-decryption --query "Parameter.Value" --out text)
+
+    - gpg_key=$(aws ssm get-parameter --region us-east-1 --name ci.jmxfetch.signing.gpg_private_key --with-decryption --query "Parameter.Value" --out text)
+    - printf -- "$gpg_key" | gpg --import --batch
```

Comment:
should we explicitly use `set +x` when manipulating these secrets? (similar to what's done in the agent pipeline: https://github.com/DataDog/datadog-agent/blob/490b0bc2a07d7d6693d5d5955de4194e286a422c/.gitlab/package_build/rpm.yml#L20-L24)

Reply:
Fixed